manager/ui/hawtio/plugins/api-manager/ts/catalog/api-catalog.ts

```diff
@@ -60,6 +60,9 @@ module Apiman {
                     if (api.endpointType == 'soap') {
                         api.ticon = 'fa-file-code-o';
                     }
+                    if (api.routeDefinitionUrl != null) {
+                        api.definitionUrl = api.routeDefinitionUrl;
+                    }
                 });
```

Comment:
So what's the point of returning both definition URLs if the routeDefinitionURL overrides the definitionURL if it exists?  E.g. when routeDefinitionURL then definitionURL isn't used (cannot be used because it is overwritten).